vendor/vendor.json

```diff
@@ -81,12 +81,6 @@
 			"revisionTime": "2015-12-06T04:24:12Z"
 		},
 		{
-			"checksumSHA1": "pWIsEUE53NffjS1goRml43Cvleg=",
```

Comment:
Aye... "fixed."  See #404 and the updated comment I made at the bottom of #386 .